catchup/catchpointService.go

```diff
@@ -459,7 +459,7 @@ func (cs *CatchpointCatchupService) processStageBlocksDownload() (err error) {
 	}
 
 	// pick the lookback with the greater of either MaxTxnLife or MaxBalLookback
-	lookback := config.Consensus[topBlock.CurrentProtocol].MaxTxnLife
+	lookback := config.Consensus[topBlock.CurrentProtocol].MaxTxnLife + 1
```

Comment:
Set `lookback` to +1 to download an extra block during catchup

Reply:
It's ok - but please add a comment in code explaining why we do so.